Gemfile

```diff
@@ -36,6 +36,8 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", require: false
 
+gem "open3", "~> 0.1.2"
```

Comment:
Can a little bit more care be taken as to where in Gemfile new gems are added please.

If comments in a previous PR are still relevant and team added gems need to be separated from pre-existing gems, then this declaration should move to the bottom of the file.